options/glibc/generic/sys-io.cpp

```diff
@@ -0,0 +1,12 @@
+#include <sys/io.h>
+#include <bits/ensure.h>
+
+int ioperm(unsigned long int, unsigned long int, int) {
+	__ensure(!"Not implemented");
+	__builtin_unreachable();
+}
+
+int iopl(int) {
+	__ensure(!"Not implemented");
+	__builtin_unreachable();
+}
```

Comment:
both of these are x86(_64) specific and syscall wrappers, stubbing them is extremely silly